src/txn/txn_recover.c

```diff
@@ -1089,6 +1089,10 @@ __wt_txn_recover(WT_SESSION_IMPL *session, const char *cfg[])
         __wt_err(session, ret, "Recovery failed");
     }
 
+    if (conn->partial_backup_remove_list != NULL) {
+        __wt_free(session, conn->partial_backup_remove_list);
+    }
```

Comment:
Style: No need for curly braces here. Probably add a comment about why this is here since no other recovery code uses this field.